src/listeners/MainListener.js

```diff
@@ -137,6 +137,7 @@ module.exports = class MainListener extends EventListener {
       const cmd = fullCmd[0].toLowerCase().trim()
       const command = this.commands.find(c => c.name.toLowerCase() === cmd || (c.aliases && c.aliases.includes(cmd)))
       if (command) {
+        if (!message.channel.permissionsFor(this.user.id).has('SEND_MESSAGES')) return
```

Comment:
I should add a message for no perms later but at least that's the thing whos gonna make the bot unbreakable when it has no perms